CMakeLists.txt

```diff
@@ -842,9 +842,12 @@ endif(WIN32)
 add_definitions(-DFEATURE_STANDALONE_SN)
 add_definitions(-DFEATURE_STRONGNAME_DELAY_SIGNING_ALLOWED)
 add_definitions(-DFEATURE_STRONGNAME_MIGRATION)
-if (CLR_CMAKE_PLATFORM_UNIX OR CLR_CMAKE_PLATFORM_ARCH_ARM64)
+if (CLR_CMAKE_PLATFORM_UNIX AND NOT CLR_CMAKE_PLATFORM_ARCH_ARM)
     add_definitions(-DFEATURE_STUBS_AS_IL)
-endif(CLR_CMAKE_PLATFORM_UNIX OR CLR_CMAKE_PLATFORM_ARCH_ARM64)
+endif (CLR_CMAKE_PLATFORM_UNIX AND NOT CLR_CMAKE_PLATFORM_ARCH_ARM)
+if (CLR_CMAKE_PLATFORM_ARCH_ARM64)
+    add_definitions(-DFEATURE_STUBS_AS_IL)
```

Comment:
That's true. I was actually wondering if I should disable FEATURE_STUBS_AS_IL for Arm64?